sdks/python/apache_beam/io/gcp/experimental/spannerio.py

```diff
@@ -284,6 +288,8 @@ class _BeamSpannerConfiguration(namedtuple("_BeamSpannerConfiguration",
                                            ["project",
                                             "instance",
                                             "database",
+                                            "table",
```

Comment:
I think this is consistent with the parameters in this namedtuple~ otherwise, there would be ```project_id``` and ```instance_id```, ```database_id```, etc. What do you think?